docs/blog/newsletter-october-2023.rst

```diff
@@ -0,0 +1,130 @@
+.. post:: October 06, 2023
+  :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – October 2023
+#########################################
+
+Ave, documentarians. Halloween may be approaching, but there's nothing scary about this month's batch of stories from the world of Write the Docs.
+
+The Write the Docs `Documentarian Salary Survey for 2023 <https://salary-survey.writethedocs.org/>`__ is open and accepting submissions! This is the survey's fifth year and, in addition to our regular salary and job-related questions, we're exploring our community's thoughts on "back to the office" mandates, feelings about job security in a volatile market, and attitudes towards pay transparency.
+
+So whether you're an employee, contractor, freelancer, or currently unemployed, `head on over <https://salary-survey.writethedocs.org/>`_ and help us make this year's survey the most useful yet.
+
+In conference news, `tickets for the in-person Australia conference </conf/australia/2023/tickets/>`__ are still open. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

Comment:
Speakers should be announced soon, right @swapnilogale ?

Reply:
Yes, just overnight if I can